oauthproxy.go

```diff
@@ -793,6 +813,16 @@ func (p *OAuthProxy) OAuthCallback(rw http.ResponseWriter, req *http.Request) {
 	}
 	errorString := req.Form.Get("error")
 	if errorString != "" {
+		errorDescription := req.Form.Get("error_description")
+
+		ceb := p.findCallbackErrorRedirect(errorString, errorDescription)
+
+		if ceb != nil {
+			logger.Printf("Configured redirect to %s", ceb.RedirectUrl)
```

Comment:
I disagree with that. The log indicates that the actual action has been recognised and that the redirect will occur (positive action). Whilst it doesn't identify which rule was matched, it can assist with fault diagnosis in a live system. I can see however that the message sucks, how about

```
			logger.Printf("Executing callback error redirect, redirecting to %s", ceb.RedirectURL)
```

Note that I spent several days getting oauth2_proxy working, this for a number of reasons, but the problem was compounded because oauth2_proxy logs incoming requests, but not outgoing requests to the identity provider. I can understand you not wanting lots of logs firing, let me know what you think, I can remove it if you doin't agree.

Reply:
Right yes, I think I follow. You would get a response in the log that says the request was redirect though right? I'd expect this line to come out and then the `ResponseLogger` to log that the response was redirected.

I wonder if we do actually want to have the location for the redirect logged here

This is the kind of situation that makes me wish we had levelled logging 😅

Reply:
Levelled logging would be really useful! I just read a blog on this https://www.honeybadger.io/blog/golang-logging/
It looks like you can easily implement wrappers for each log level (which seems obvious) and use the correct wrapper for each error level. I think that since logs have to go out on STDOUT/STDERR for Kubernetes/Docker, anything cleverer than this would just be a waste of effort.